### PR TITLE
add demo files to npmignore

### PR DESCRIPTION
**Pull request by tech4GT**
Signed-off-by: Varun Gupta <varun.gupta1798@gmail.com>

### CODE REVIEW

**Comment by jywarren** [comment]
Ah, this is good, but sometimes I wonder if it's good to have the docs included even in the npm distribution? But are they too big and are really taking up space? If so, then I agree with you. :-)

**Comment by tech4GT** [comment]
I think we should not package the demo inside the npm package since there is no use for it there anyway, don't you think?

**Comment by tech4GT** [comment]
Do you mean the contributing docs?

**Comment by jywarren** [comment]
Sorry i meant the demo itself. If it's really just a few files, I think it's worth keeping in the distributed version. But if it weighs a lot, we can remove them. What do you think? Sometimes it's really helpful to see a functioning demo of the code, even in a distribution... but maybe this kind of violates the convention?

**Comment by tech4GT** [comment]
@jywarren Yeah it can, but most people who will be downloading this via npm would be using this in their own projects or using the cli, so I think we can safely remove the demo and dist files from npm, what say?

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+*.swp
+todo.txt
+test.js
+output.txt
+output/
+
+examples/
+icons/
+dist/
```
Comment (jywarren) [inline .npmignore]: oh but not dist, right?

Comment (jywarren) [reply .npmignore]: and test?

Comment (tech4GT) [reply .npmignore]: Oh yeah, I'll add test! 
Do you think we need dist in the npm distribution, I was thinking maybe we can distribute the dist file separately using a cdn, kind of like jquery, what say?

**Comment by tech4GT** [comment]
Okay, adding tests and dist from npmignore!

**Pull request merged**